showcase/src/assets/translations/fr.json

```diff
@@ -109,6 +109,28 @@
         "SIMPLE_HORIZONTAL_SLIDER": "Curseur simple",
         "SIMPLE_VERTICAL_SLIDER": "Curseur simple",
         "VERTICAL_RANGE_SLIDER": "Curseur de plage"
+      },
+      "TYPOGRAPHY": {
```

Comment:
Could you order these translations in alphabetical order so they are easier to locate?